src/main/java/org/jabref/logic/openoffice/style/OOBibStyle.java

```diff
@@ -157,11 +183,24 @@ private void setDefaultProperties() {
         properties.put(IS_NUMBER_ENTRIES, Boolean.FALSE);
         properties.put(BRACKET_BEFORE, "[");
         properties.put(BRACKET_AFTER, "]");
-        properties.put(REFERENCE_PARAGRAPH_FORMAT, "Default");
+        properties.put(REFERENCE_PARAGRAPH_FORMAT, "Standard");
```

Comment:
https://wiki.openoffice.org/wiki/Documentation/DevGuide/Text/Overall_Document_Features#Styles
https://wiki.openoffice.org/wiki/API/Implementation_Details/Programmatic_StyleNames
[How To Obtain Default PAra Style Name](https://forum.openoffice.org/en/forum/viewtopic.php?f=45&t=77830)

> Is there a reason to change this?

Styles have two names: one for programming (`getName()`, `setName()`) and one for display purposes `DisplayName`. The DisplayName is localized, meaning it can differ between locales. `"Standard"` is the programmatic (locale-independent) name for the paragraph style with DisplayName `"Default"` in some locales. The programmatic names work even if the locale used differs from that used by the author of a jstyle (or in this case jabref). 

> Won't break this backwards compatibility?

It should improve compatibility across locales and possible changes of DisplayName.

Reply:
In [OOBibBase2.java/checkStyleExistsInTheDocument](https://github.com/antalk2/jabref/blob/835e8f1bcc38316deb781d6cffc28c1ccb5641d6/src/main/java/org/jabref/gui/openoffice/OOBibBase2.java#L398)
I inserted a warning against non-programmatic style names,
to warn about locale-dependent style names in jstyle files..
That pointed out these builtin default values as well. 
It will also warn, I believe about

File: src/main/resources/resource/openoffice/default_authoryear.jstyle
39:25:CitationCharacterFormat="Default"

File: src/main/resources/resource/openoffice/default_numerical.jstyle
27:25:CitationCharacterFormat="Default"

and some styles at jstyles.jabref.org-master